src/main/java/com/globallogic/amcr/model/wiki/Wiki.java

```diff
@@ -0,0 +1,141 @@
+package com.globallogic.amcr.model.wiki;
+
+import com.globallogic.amcr.utils.Assert;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.UUID;
+
+@SuppressWarnings("unused")
+public class Wiki {
+
+    private UUID id;
+    @NotNull
+    private String title;
+    @NotNull
+    private String overview;
+
+    private UUID[] subImage;
+    @NotNull
+    private String subTitle;
```

Comment:
What are subtitle & suboverview for

Reply:
From checking the Figma, its mainly for the section near the image on the wiki page single

Reply:
Sounds good, maybe some javadocs to make that obvious could be useful

Reply:
Sorry, it looks like you have them below but it's not too descriptive, remember to auto format too before committing

Reply:
yeah reformatted them now (IntelliJ can do it on save now) and expanded on the descriptions